classes/article/ArticleGalleyDAO.inc.php

```diff
@@ -235,27 +248,49 @@ function _fromRow($row) {
 	function insertObject($galley) {
 		$this->update(
 			'INSERT INTO submission_galleys
-				(submission_id, label, locale, seq, remote_url, file_id)
+				(submission_id, label, locale, seq, remote_url)
 				VALUES
-				(?, ?, ?, ?, ?, ?)',
+				(?, ?, ?, ?, ?)',
 			array(
 				(int) $galley->getSubmissionId(),
 				$galley->getLabel(),
 				$galley->getLocale(),
 				$galley->getSequence() == null ? $this->getNextGalleySequence($galley->getSubmissionId()) : $galley->getSequence(),
 				$galley->getRemoteURL(),
-				$galley->getFileId(),
 			)
 		);
 		$galley->setId($this->getInsertId());
 		$this->updateLocaleFields($galley);
+		$this->addFile($galley->getId(), $galley->getFileId(), $galley->getCurrentVersionId());
 
 		HookRegistry::call('ArticleGalleyDAO::insertNewGalley', array(&$galley, $galley->getId()));
 
 		return $galley->getId();
 	}
 
 	/**
+	* Connect galley version with file in submission_galley_files
+	* @param $galleyId ArticleGalleyId
```

Comment:
The syntax for parameter is @param variable_name type explanation, thus here and below:
@param $galleyId integer ...
This ways it looks like ArticleGalleyId, FileId and SubmissionRevision are objects/classes